test/Microsoft.ML.Tests/Scenarios/Api/TestApi.cs

```diff
@@ -402,12 +402,13 @@ public void TestTrainTestSplitWithStratification()
                 },
             });
 
+            // TEST TRAINTESTSPLIT
             var split = mlContext.Data.TrainTestSplit(input, 0.5, nameof(Input.TextStrat));
             var ids = split.TestSet.GetColumn<int>(split.TestSet.Schema[nameof(Input.Id)]);
             Assert.Contains(1, ids);
             Assert.Contains(5, ids);
             split = mlContext.Data.TrainTestSplit(input, 0.5, nameof(Input.FloatStrat));
-            ids = split.TrainSet.GetColumn<int>(split.TrainSet.Schema[nameof(Input.Id)]);
+            ids = split.TestSet.GetColumn<int>(split.TestSet.Schema[nameof(Input.Id)]);
```

Comment:
>TestSet [](start = 24, length = 7)

why?

Reply:
Because of the change to `ensureZeroUntouched = false`

---
In reply to: [443437139](https://github.com/dotnet/machinelearning/pull/5227#discussion_r443437139) [](ancestors = 443437139)